_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+- A reviewer should not evaluate a manuscript authored or co-authored by a person with whom the reviewer has a personal or professional connection if the relationship would  bias judgment of the manuscript, or reasonably be assumed to bias judgment
+- Confidentiality and peer reviewer anonymity are expectations throughout the editorial review process in order to allow for candid discussion and evaluation regarding submitted scientific content. This expectation of peer review confidentiality is independent of, and extends beyond, the final decision on the manuscript (i.e., whether or not the manuscript was published or rejected). More specifically: 
+  - A reviewer should treat both the submitted manuscript and data received from the journal, and his/her referee report and related correspondence as confidential documents. Such documents and their contents should neither be disclosed to nor discussed with others. In addition, the reviewer may not disclose the contents of the submitted review to any individual or organization. 
+  - For educational purposes, the reviewer may wish to enlist the assistance of a trainee currently under their direction (for example, a Ph.D. student or postdoctoral researcher advised by the reviewer) in reviewing the manuscript. This practice is allowed under the stipulations that the assigned reviewer (i) clearly communicate the need for confidentiality to the trainee (ii) must read and approve the review report prior to submission, and (iii) must disclose to the editor the name of the trainee that provided the assistance.
+  - In some special cases, the assigned reviewer may desire some level of input, advice, or guidance from a specific expert to facilitate review of the manuscript. In such instances, the reviewer must seek approval from the editor prior to sharing the contents of the manuscript with the expert. Following editorial approval, the assigned reviewer must clearly communicate the need for confidentiality to the consulted expert. The identity of the expert must also be disclosed to the editor at that time.  
+  - In some special cases, the reviewer may wish to disclose his/her identity to the manuscript authors, the editor should be consulted. There may be instances where the editor requests that the reviewer retains anonymity. The decision to disclose such information to authors is the sole discretion of the editor. However, such decisions should be accompanied by appropriate rationale from the editor. 
+  - Reviewers may disclose publicly that they have served as a reviewer for LiveCoMS. However, under no circumstances should the reviewer identify himself/herself as the reviewer of a specific manuscript. 
```

Comment:
Same comment as above. Also, this would preclude anyone doing open commenting on GitHub in ways that could reveal they had reviewed the paper, whereas I think this is something we'd like to see happen!